Cesium.Runtime/Cesium.Runtime.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net6.0</TargetFrameworks>
+    <TargetFramework>net7.0</TargetFramework>
```

Comment:
I would still insist on having netstandard2.0 support via `#if`.

Reply:
Hmm, this is runtime. Is it planned to be referenced as a package by .NET Framework/pre-.NET 7 targets or is it packaged as a standalone, well, runtime?

Reply:
Yes, it is planned to be referenced by projects targeting older .NET versions.